code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -123,8 +123,9 @@
 		if(!just_sleeping)
 			if(suiciding)
 				. += "<span class='warning'>[t_He] appear[p_s()] to have committed suicide... there is no hope of recovery.</span>"
-
-			. += generate_death_examine_text()
+			. += ""
```

Comment:
Why add nothing to the list?

Reply:
Because that's exactly how it used to be, this PR is just a revert but the nice revert button is broken. I can probably stare into the details more after a little bit of testing.

Reply:
Wouldn't the normal text be "They are limp and unresponsive; there are no signs of life."

Reply:
which is what happens, provided they don't disconnect or DNR or any other reason they are locked out of their body.

Reply:
How though? I don't see any code reachable that emits the regular death message.

Reply:
I tested it repeatedly and it works, my knowledge of code is fairly limited so I cannot explain why or how it works.

Reply:
This works but there's no need for it.
It's safe to remove.
If it was superfluous before and it was removed by a PR, it's not justified to re-add it without a good reason.